Creiwry/irl-meeting-app
# Add string methods for user model

### Ticket Link
https://templatethpprojetfinal.atlassian.net/browse/THPPROJ-57?atlOrigin=eyJpIjoiY2Q2OWM4OGY3ZTgyNDllMWIxNWUwZWEzODgwODJiNDAiLCJwIjoiaiJ9

### What does this PR do?
It adds the methods gender_string, sexual_orientation_string, and relationship_status_string to the User model. These methods map and return a string value for the integer value represented by the relevant attribute (gender, sexual_orientation, relationship_status).
This PR also adds tests for these methods.

### Where should the reviewer start?
All the changes to code were done in the User model.

### How should this be manually tested?
There are tests for these methods that pass. However, the user can run a seed, and then do user.gender_string, user.sexual_orientation and user.relationship_status to see string outputs

### Are there any breaking changes?
No
